packages/cli/commands/project/listBuilds.js

```diff
@@ -0,0 +1,173 @@
+const path = require('path');
+
+const {
+  addAccountOptions,
+  addConfigOptions,
+  setLogLevel,
+  getAccountId,
+  addUseEnvironmentOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cli-lib');
+const {
+  logApiErrorInstance,
+  ApiErrorContext,
+} = require('@hubspot/cli-lib/errorHandlers');
+const { logger } = require('@hubspot/cli-lib/logger');
+const {
+  fetchProject,
+  fetchProjectBuilds,
+} = require('@hubspot/cli-lib/api/dfs');
+const {
+  getTableContents,
+  getTableHeader,
+} = require('@hubspot/cli-lib/lib/table');
+const { getCwd } = require('@hubspot/cli-lib/path');
+const { validateAccount } = require('../../lib/validation');
+const { link } = require('../../lib/links');
+const { getProjectConfig, getProjectDetailUrl } = require('../../lib/projects');
+const moment = require('moment');
+const { prompt } = require('inquirer');
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validateAccount(options)))) {
+    process.exit(1);
+  }
+};
+
+exports.command = 'list-builds [path]';
+exports.describe = false;
+
+exports.handler = async options => {
+  loadAndValidateOptions(options);
+
+  const { path: projectPath, limit } = options;
+  const accountId = getAccountId(options);
+
+  trackCommandUsage('project-list-builds', { projectPath }, accountId);
+
+  const cwd = projectPath ? path.resolve(getCwd(), projectPath) : getCwd();
+  const projectConfig = await getProjectConfig(cwd);
```

Comment:
We should do a check for projectConfig here. I ran `hs project list-builds some/bad/path` and I get an error that says `Cannot read property 'name' of null` coming from [this line here](https://github.com/HubSpot/hubspot-cli/pull/582/files#diff-36a35fb191a893b8f5de180e0cc2bd7656d4c00c972ef65a9687a6e3f8304817R136).

Maybe we could wrap up the error handling for missing project config into the `getProjectConfig` util? So all the commands that use it will get the error handling for free.

Reply:
Ah, that's a good call and a good idea. Will do

Reply:
I forgot we have `validateProjectConfig()` for this. Added here: https://github.com/HubSpot/hubspot-cli/pull/582/files#diff-36a35fb191a893b8f5de180e0cc2bd7656d4c00c972ef65a9687a6e3f8304817R67